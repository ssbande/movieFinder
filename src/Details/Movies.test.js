import React from 'react';
import { shallow } from 'enzyme';

import Movie from './Movies';

const setup = () => ({
  props: {
    getDetails: jest.fn(),
    history: { push: jest.fn() },
    category: 'movie',
    detailInfo: {
      id: 1001,
      profile_path: 'profile_path',
      title: 'title',
      overview: 'overview',
      genres: [{ name: 'genre1' }, { name: 'genre2' }, { name: 'genre3' }]
    }
  }
})

describe('Movies page tests', () => {
  it('should render correctly', () => {
    const { props } = setup();
    const wrapper = shallow(<Movie {...props} />);
    expect(wrapper).toMatchSnapshot();
  })
})