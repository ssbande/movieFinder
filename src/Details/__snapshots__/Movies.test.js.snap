// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Movies page tests should render correctly 1`] = `ShallowWrapper {}`;
